src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -109,7 +132,7 @@
 					<PropertyGroup>
 						<CodesignEntitlements>./iOS/Entitlements.plist</CodesignEntitlements>
 						<MtouchDebug>True</MtouchDebug>
-						<MtouchExtraArgs>$(MtouchExtraArgs) --optimize=-remove-dynamic-registrar --setenv=MONO_LOG_LEVEL=debug --setenv=MONO_LOG_MASK=gc</MtouchExtraArgs>
```

Comment:
Should this be changing? Is it related to splash screen?

Reply:
No I was getting a warning saying that this was ignored because we are using linking.

Reply:
but did the warning start appearing with the splashscreen?

Reply:
No, just noticed it since I rebuild the mobile head.